nnsearch/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "nnsearch.fullname" . }}
+  labels:
+    app: {{ template "nnsearch.name" . }}
+    chart: {{ template "nnsearch.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
```

Comment:
This should be 1323 and be taken from values.yaml

Reply:
Done